.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
```

Comment:
Why did you decide to split up the building pipeline into separate jobs? In that case, the actual building of your application would have to be done inside each job.

Reply:
> Why did you decide to split up the building pipeline into separate jobs? In that case, the actual building of your application would have to be done inside each job.

now, there is one job with several steps, but you said that we would not be punished for that comment due to the fact that we had been confused with your answer about jobs at the beginning.